src/components/BulkEditLogs/BulkEditLogs.js

```diff
@@ -16,24 +16,29 @@ import {
 
 import { LOGS_COLUMNS } from '../../constants';
 import { getLogsResultsFormatter } from '../../utils/formatters';
-import { useLogsQueryParams } from '../../hooks';
+import { useBulkPermissions, useLogsQueryParams } from '../../hooks';
 import { useBulkEditLogs } from '../../hooks/api';
 
 const resetData = () => {};
 
-const visibleColumns = LOGS_COLUMNS.map(i => i.value);
+const getVisibleColumns = (perms) => LOGS_COLUMNS
+  .filter(col => perms.hasUsersViewPerms || col.value !== 'actions')
```

Comment:
you need to hide action and not column, if user doesn't have users permissions it doesn't mean that Items files should be restricted for download

BulkEditLogsActions is the place where modifications should be done

Reply:
but what is the reason show the column `actions` without any actions inside?